ballerina/build.gradle

```
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+buildscript {
+    repositories {
+        maven {
+            url = 'https://maven.pkg.github.com/ballerina-platform/plugin-gradle'
+            credentials {
+                username System.getenv("packageUser")
+                password System.getenv("packagePAT")
+            }
+        }
+    }
+    dependencies {
+        classpath "io.ballerina:plugin-gradle:${project.ballerinaGradlePluginVersion}"
+    }
+}
+
+description = 'Ballerina - Observability'
+
+def packageName = "observe"
+def packageOrg = "ballerina"
+
+def tomlVersion = stripBallerinaExtensionVersion("${project.version}")
+def ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/Ballerina.toml")
+def ballerinaTomlFile = new File("$project.projectDir/Ballerina.toml")
+
+def stripBallerinaExtensionVersion(String extVersion) {
+    if (extVersion.matches(project.ext.timestampedVersionRegex)) {
+        def splitVersion = extVersion.split('-');
+        if (splitVersion.length > 3) {
+            def strippedValues = splitVersion[0..-4]
+            return strippedValues.join('-')
+        } else {
+            return extVersion
+        }
+    } else {
+        return extVersion.replace("${project.ext.snapshotVersion}", "")
+    }
+}
+
+apply plugin: 'io.ballerina.plugin'
+
+ballerina {
+    packageOrganization = packageOrg
+    module = packageName
+    langVersion = ballerinaLangVersion
+}
+
+configurations {
+    externalJars
+}
+
+dependencies {
+    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-api', version: "${openTelemetryVersion}") {
+        transitive = false
+    }
+    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-sdk-trace', version: "${openTelemetryVersion}") {
+        transitive = false
+    }
+    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-sdk-testing', version: "${openTelemetryVersion}") {
+        transitive = false
+    }
+    externalJars(group: 'org.testng', name: 'testng', version: "${testngVersion}") {
+        transitive = false
+    }
+}
```

We can simply specify `transitive = false` once
```suggestion
configurations {
    externalJars {
        transitive = false
    }
}

dependencies {
    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-api', version: "${openTelemetryVersion}")
    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-sdk-trace', version: "${openTelemetryVersion}")
    externalJars(group: 'io.opentelemetry', name: 'opentelemetry-sdk-testing', version: "${openTelemetryVersion}")
    externalJars(group: 'org.testng', name: 'testng', version: "${testngVersion}")
}
```

---

Please fix in other relevant places as well.

---

Fixed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/files#diff-54c2a1c9d62367d38139ae3d5d8200e06bc932ab7905f7e666d2c895382b2b81